transifex/native/parsing.py

```diff
@@ -43,6 +43,18 @@ def __init__(self, string, _context=None, **meta):
         )  # type: list
         self.meta = self._transform_meta(meta)
 
+    @property
+    def occurrence(self):
+        """An optional occurrence field, shows were the string is located
+
+        :rtype: unicode
+        """
+        return self.meta.get(consts.KEY_OCCURRENCE)
+
+    @occurrence.setter
+    def occurrence(self, value):
+        self.meta[consts.KEY_OCCURRENCE] = value
```

Comment:
@candrikos how about we name it `occurrences` and make it a list? Just like `tags`. I don't like that we keep it serialized in the model. We can later serialized it when sending it to CDS.

Reply:
@dbendilas I can do that, why keep serialized things is something you do not prefer?

Reply:
Well, I think of it as data and it makes sense to me to have the data in a model ready for consumption. Simply put, in reality it's a list, so let's make it a list, just like we keep `dict`s instead of their JSON (or other) serialization in a model like this. Keeping the occurrences serialized with `;` is something very "custom" and I don't see any technical (or otherwise) restriction to make us do this. 

In contrast, we would have to do that for JSON structures in older versions of some DBMSs, so there wouldn't be much of a choice there.

Reply:
@dbendilas I updated that but I think that occurrences are not going to be consumed thus there is no reason to maintain them in a non-serialized format The big picture regarding the SDK is to extract push and pull SourceStrings, there is no actual reason for such high-level modelling.